prereise/gather/demanddata/transportation_electrification/generate_BEV_vehicle_profiles.py

```diff
@@ -0,0 +1,146 @@
+import os
+
+import pandas as pd
+
+from prereise.gather.demanddata.transportation_electrification import (
+    const,
+    immediate,
+    smart_charging,
+)
+from prereise.gather.demanddata.transportation_electrification.data_helper import (
+    generate_daily_weighting,
+    get_kwhmi,
+    load_rural_scaling_factor,
+    load_urbanized_scaling_factor,
+)
+
+
+def generate_bev_vehicle_profiles(
+    vehicle_trip_data_filepath,
+    charging_strategy,
+    veh_type,
+    veh_range,
+    model_year,
+    state,
+    load_demand,
+    power=6.6,
+    location_strategy=2,
+    trip_strategy=1,
+):
+    """Generate Battery Electric Vehicle (BEV) profiles
+
+    :param str vehicle_trip_data_filepath: collected trip data from external sources representing driving patterns
+    :param str charging_strategy: establishes whether charging happens immediately or via a smart charging algorithm;
+                current options are "immediate" and "smart"
+    :param str veh_type: vehicle category (LDV, LDT, MDV, HDV)
+    :param int veh_range: 100, 200, or 300, represents how far vehicle can travel on
+        single charge.
+    :param int model_year: year that is being modelled/projected to, 2017, 2030, 2040,
+        2050.
+    :param str state: US state abbreviation
+    :param np.array load_demand: initial load demand (MW for each hour)
+    :param int power: (optional) charger power, EVSE kW; default value: 6.6 kW;
+    :param int location_strategy: (optional) where the vehicle can charge-1, 2, 3, 4, or 5;
+        1-home only, 2-home and work related, 3-anywhere if possibile,
```

Comment:
Nothing as far as I know.  I'll double-check with Kate.

Reply:
Aha!  Reply from Kate: `It is currently labeled work-related, because there are a couple "whyfrom" and "whyto" answers that have work in the description.`  For instance, `04=Work-related meeting / trip`.  This is a nuance that I don't think is worth parsing in the data nor the code.  I'm ok with leaving it as `work related` but also open to changing it.